common/djangoapps/student/models.py

```diff
@@ -266,15 +266,17 @@ class Meta(object):
 
     user_age = models.CharField(max_length=3, db_index=True, choices=AGE_CHOICES, default=CHOICE_UNDISCLOSED)
 
-    GENDER_OTHER = 'o'
+    GENDER_UNDISCLOSED = 'u'
     GENDER_CHOICES = (
-        ('m', ugettext_noop('Male')),
         ('f', ugettext_noop('Female')),
+        ('g', ugettext_noop('Gender Nonconforming')),
+        ('m', ugettext_noop('Male')),
+        ('o', ugettext_noop('Other')),
```

Comment:
Are you sure we add gender `Other`?

Reply:
Now we need this choices: `Female / Gender Nonconforming / Male / Other / Undisclosed`